helpwave/web
# fix: replace edit with edit icon

This pull request replace the edit text on my organization and the ward page with an edit icon.

Closes #174 Replace edit with an edit icon.

Testing instructions:
1. Install all the required packages from the package.json file.
2. Go to my organization page and notice the edit text is replaced with an edit icon.
3. Go to the ward page and notice the edit text is replaced with an edit icon.

Additional notes:
- Added one extra package that is react-icons that must be installed before use.
